frontend/src/routes/chat/Chat.svelte

```diff
@@ -1,79 +0,0 @@
-<script lang="ts">
```

Comment:
It's ok to delete this but we will need to refer to it in the future for the css

Reply:
fixed